pkg/antctl/raw/helper.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package raw
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	agentapiserver "antrea.io/antrea/pkg/agent/apiserver"
+	"antrea.io/antrea/pkg/antctl/runtime"
+	"antrea.io/antrea/pkg/apis"
+	clusterinformationv1beta1 "antrea.io/antrea/pkg/apis/crd/v1beta1"
+	controllerapiserver "antrea.io/antrea/pkg/apiserver"
+	antrea "antrea.io/antrea/pkg/client/clientset/versioned"
+	"antrea.io/antrea/pkg/client/clientset/versioned/scheme"
+	"antrea.io/antrea/pkg/util/k8s"
+)
+
+func SetupClients(kubeconfig *rest.Config) (*kubernetes.Clientset, *antrea.Clientset, error) {
+	k8sClientset, err := kubernetes.NewForConfig(kubeconfig)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to create K8s clientset: %w", err)
+	}
+	antreaClientset, err := antrea.NewForConfig(kubeconfig)
+	if err != nil {
+		return k8sClientset, nil, fmt.Errorf("error when creating Antrea clientset: %w", err)
+	}
+	return k8sClientset, antreaClientset, nil
+}
+
+func ResolveKubeconfig(cmd *cobra.Command) (*rest.Config, error) {
+	kubeconfigPath, err := cmd.Flags().GetString("kubeconfig")
+	if err != nil {
+		return nil, err
+	}
+	kubeconfig, err := runtime.ResolveKubeconfig(kubeconfigPath)
+	if err != nil {
+		return nil, err
+	}
+	return kubeconfig, nil
+}
+
+// TODO: enable secure connection.
+func SetupInsecureKubeconfig(kubeconfig *rest.Config) {
+	kubeconfig.Insecure = true
+	kubeconfig.CAFile = ""
+	kubeconfig.CAData = nil
+}
```

Comment:
This function may be confusing. There is one `SetupKubeconfig` too so looks like that's a secure one and this is insecure one. Can't it caller use `SetupKubeconfig`?

Reply:
just refined it with name SetupBaseKubeconfig, this simple one will be called in proxy command. we haven't had a secure one yet.

Reply:
ah, I may misunderstood it, let me check if it's possible to just use SetupKubeconfig for proxy command

Reply:
after tried it locally, proxy command can work with SetupKubeconfig, so I just removed SetupBaseKubeconfig.